torch/_meta_registrations.py

```diff
@@ -2575,6 +2575,11 @@ def mkldnn_rnn_layer_backward(
     return diff_x, diff_w1, diff_w2, diff_b, diff_b, diff_hx, diff_cx
 
 
+@register_meta(aten.bucketize)
```

Comment:
This impl only works for aten.bucketize.Tensor, and won't work for aten.bucketize.Scalar.

as torch.empty_like(scalar) doesn't work.... 

You might want to update it to `@register_meta(aten.bucketize.Tensor)`